.github/workflows/continuous-integration.yml

```
@@ -29,12 +25,26 @@ jobs:
       run: python -m pip install --upgrade pip -r test-requirements.txt
     - name: Run pytest
       run: pytest
+  check-style:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
     - name: Check style
-      run: flake8
+      uses: py-actions/flake8@v2
+  check-code-formatting:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
     - name: Check code formatting
-      run: black --check --diff .
+      uses: psf/black@stable
+  code-cov:
```

```suggestion
  code-cov:
    needs: python-tests
```

Otherwise the coverage results are not available.